### DISCUSSION TITLE
2fa error ubuntu

### DISCUSSION THREAD

**Discussion by ch0ppe4** [RakambdaOrg/ChannelPointsMiner#336] [Q&A] [answered]
![screenshot 2022-10-26 44](https://user-images.githubusercontent.com/116724267/198010674-81756f1e-a2b8-430b-89c4-2ee0621bbb0f.png)
Can u help me?

**Comment by ch0ppe4** [comment]
![изображение](https://user-images.githubusercontent.com/116724267/198010825-c0221b13-7f10-4aef-8c29-c3e3e2a5694f.png)
Can u add me in discord?
My tag золотце#0790

**Reply by Rakambda** [reply]
Join the public discord server instead if you want help

**Reply by ch0ppe4** [reply]
link?

**Reply by Rakambda** [reply]
In the readme at the top

**Comment by Rakambda** [comment, marked as answer]
[`http` login method isn't working due to changes by twitch, use `mobile` instead.](https://raksrinana.github.io/ChannelPointsMiner/miner/main/configuration/global.html#_login_method)
